crates/orderbook/src/database/orders.rs

```diff
@@ -854,69 +841,6 @@ mod tests {
         );
     }
 
-    #[tokio::test]
```

Comment:
I felt this didn't test anything useful because I moved the quotes roundtrip test into the database crate. The only thing extra this tests is that the function in orderbook insert_order really forwards to database insert_order and insert_quote.